### ISSUE TITLE
Return IlpReject packet if _handleCustomData throws an error

### ISSUE THREAD

**Issue by emschwartz** [interledgerjs/ilp-plugin-mini-accounts#22]
Right now it seems like nothing will be returned if _handleCustomData throws an error. It should at least respond with a `T00` or something like that, and maybe include the error message.

**Comment by sharafian** [comment]
I don't think that behavior makes sense, because `_handleCustomData` handles all types of incoming data, not just ILP. I implemented this error handing inside of asym-server instead, by wrapping the ILP packet handling logic in a try catch that returns reject packets.

**Comment by emschwartz** [comment]
Would it make more sense for mini accounts to separate out ILP packets from other things? The problem I see is that you've now implemented https://github.com/interledgerjs/ilp-plugin-xrp-asym-server/pull/22 but the same needs to be done for the other plugins that extend mini accounts also

**Comment by sharafian** [comment]
The overhead of adding a try-catch is probably smaller than splitting data-handling logic into two handlers. Not every subclass handles ilp packets the same, either
